docs/PowerAuth-SDK-for-Android.md

```diff
@@ -1374,6 +1375,28 @@ tokenStore.removeAllLocalTokens(context);
 
 Note that by removing tokens locally, you will lose control of the tokens stored on the server.
 
+## External Encryption Key
```

Comment:
There are quite a few grammar errors (reported by the Grammarly)

Reply:
This is strange, because I was using grammarly for check and found just passive voice. Now I can see a more errors :)